Translations/it.po

```diff
@@ -3610,680 +3598,676 @@ msgstr "Amuleto del Chierico"
 msgid "Gladiator's Ring"
 msgstr "Anello del Gladiatore"
 
-#: Source/items.cpp:162
+#: Source/items.cpp:163
 msgid "Oil of Mastery"
 msgstr "Olio della Maestria"
 
-#: Source/items.cpp:164
+#: Source/items.cpp:165
 msgid "Oil of Death"
 msgstr "Olio della Morte"
 
-#: Source/items.cpp:165
+#: Source/items.cpp:166
 msgid "Oil of Skill"
 msgstr "Olio dell'Abilità"
 
-#: Source/items.cpp:167
+#: Source/items.cpp:168
 msgid "Oil of Fortitude"
 msgstr "Olio della Fortezza"
 
-#: Source/items.cpp:168
+#: Source/items.cpp:169
 msgid "Oil of Permanence"
 msgstr "Olio della Permanenza"
 
-#: Source/items.cpp:169
+#: Source/items.cpp:170
 msgid "Oil of Hardening"
 msgstr "Olio della Tempra"
 
-#: Source/items.cpp:170
+#: Source/items.cpp:171
 msgid "Oil of Imperviousness"
 msgstr "Olio dell'Impermeabilità"
 
 #. TRANSLATORS: Constructs item names. Format: {Item} of {Spell}. Example: War Staff of Firewall
-#: Source/items.cpp:1183
-#, fuzzy
-#| msgid "{0} of {1}"
+#: Source/items.cpp:1184
 msgctxt "spell"
 msgid "{0} of {1}"
-msgstr "{:s} di {:s}"
+msgstr "{0} di {1}"
 
 #. TRANSLATORS: Constructs item names. Format: {Prefix} {Item} of {Spell}. Example: King's War Staff of Firewall
-#: Source/items.cpp:1193
-#, fuzzy
+#: Source/items.cpp:1192
 msgctxt "spell"
 msgid "{0} {1} of {2}"
-msgstr "{:s} {:s} di {:s}"
+msgstr "{1} {0} di {2}"
```

Comment:
glad to see this was useful :)

Reply:
Yes. Have to refine some things. Italian also have grammatical gender for prepositions. an example:
the arrival of the train - train is male so of = del (l'arrivo del treno)
the doors of perception - perception is female so of = della (le porte della percezione).
I'm trying to adapt wherever is possible. But have to deeply dive into testing

Reply:
im currently working on a solution for grammatical genders

Reply:
You could make the prepositions part of the suffix

`{1} {0} {2}`

```
di ferro
rossa
d'angelo
del soldato
```

```
Ascia
Scudo
```

```
del massacro
di deviazione
della fragilità
dei giganti
dello stolto
della luna
delle stelle
dell'armonia
```

I know this dosent solve the issue of gendered prefixes, but I think it's the best you can currently do.